### ISSUE TITLE
tissue inconsistency - human brain

### ISSUE THREAD

**Issue by jahilton** [chanzuckerberg/single-cell-curation#630]
[283d65eb-dd53-496d-adb7-7570c7caa443](https://cellxgene.cziscience.com/collections/283d65eb-dd53-496d-adb7-7570c7caa443) & [4216ddce-94c0-4fdc-9eba-6dda2e2205f8](https://cellxgene.cziscience.com/collections/4216ddce-94c0-4fdc-9eba-6dda2e2205f8) contain overlapping data (donor_ids H18.30.001 and H18.30.002)

The latter is all primary motor cortex but there isn’t any of that tissue in the former. Need to understand which data overlaps, what the correct tissue value should be and then revise one/both accordingly.

**Comment by jahilton** [comment]
Root issue here is lack of specificity throughout [283d65eb-dd53-496d-adb7-7570c7caa443](https://cellxgene.cziscience.com/collections/283d65eb-dd53-496d-adb7-7570c7caa443)
For example, the overlapping data described in the description is in reference to the primary motor cortex. This should be the dataset titled `Dissection: Cerebral cortex (Cx) - Precentral gyrus (PrCG) - Primary motor cortex - M1C` (with the M1C == primary motor cortex)
Recently-submitted [CrossAreal study](https://cellxgene.cziscience.com/collections/d17249d2-0e6e-4500-abb8-e6c93fa1ac6f) can help annotate as it has similar tissues/abbreviations

**Comment by jahilton** [comment]
Got proposed tissue terms to contributors for review

**Comment by jahilton** [comment]
No update so poked the contributors again

**Comment by dosumis** [comment]
Hi @patrick-lloyd-ray  Would you be able to find someone at the Allen to review this mapping for @jahilton?  Thanks!

**Comment by patrick-lloyd-ray** [comment]
> Hi @patrick-lloyd-ray Would you be able to find someone at the Allen to review this mapping for @jahilton? Thanks!
> 
> https://docs.google.com/spreadsheets/d/1ZMlQxdpLwFyTBV-puqS-6056p0QUEz9ypVPYoIBXX0M/edit?gid=292822203#gid=292822203

Yeah, let's see if we can get some eyes on it. @rightbower @abhandiwad are you able to take a quick look?

**Comment by jahilton** [comment]
Went ahead and pushed the tissue updates. Those are now published
